src/ARSamples/ARToolkit.Samples.Maui/Samples/ContinuousGPSSample.xaml.cs

```diff
@@ -1,56 +1,90 @@
             json = json.Replace("{\"wkid\":4326}", "{\"wkid\":6318}");
+            if (Esri.ArcGISRuntime.Data.FeatureCollection.FromJson(json) is FeatureCollection fc)
+            {
+                var fl = new FeatureCollectionLayer(fc);
+                scene.OperationalLayers.Add(fl);
+            }
 
-            var fc = Esri.ArcGISRuntime.Data.FeatureCollection.FromJson(json);
-            var fl = new FeatureCollectionLayer(fc);
-            Scene.OperationalLayers.Add(fl);
-            Scene.OperationalLayers.Add(new FeatureLayer(new Uri("https://services5.arcgis.com/N82JbI5EYtAkuUKU/arcgis/rest/services/Fire_Hydrants_AR_Demo/FeatureServer/0")));
-
-            await Scene.LoadAsync();
-            ARView.Scene = Scene;
-            ARView.LocationDataSource = LocationDataSource.CreateDefault();
-            await ARView.StartTrackingAsync(Esri.ArcGISRuntime.ARToolkit.ARLocationTrackingMode.Continuous);
+            ARView!.TranslationFactor = 1;
+            ARView!.Scene = scene;
         }
         catch (Exception ex)
         {
-            await DisplayAlert("Failed to load scene", ex.Message, "OK");
-            await Navigation.PopAsync();
+            System.Diagnostics.Debug.WriteLine(ex);
         }
     }
 
-    protected override void OnAppearing()
+    private async Task LoadWhenReady()
+    {
+        bool hasLoaded = false;
+        do
+        {
+            try
+            {
+                await ARView.StartTrackingAsync(Esri.ArcGISRuntime.ARToolkit.ARLocationTrackingMode.Continuous);
+                hasLoaded = true;
+            }
+            catch (Exception)
+            {
+                await Task.Delay(300);
+            }
```

Comment:
This doesn't seem right. If we need to do this, we need to address the issue this is trying to work around.